content/contribute/index.md

```diff
@@ -40,94 +42,110 @@ The biggest and most active repositories we have today are:
+repositories across the Github organizations
+[IPFS](https://github.com/ipfs),
+[libp2p](https://github.com/libp2p),
+[IPLD](https://github.com/libp2p), and
+[Multiformats](https://github.com/multiformats).
+
 
 ### Documentation
 
 Again, IPFS is a huge project and undertaking. With lots of code comes the need
-of having lots of good documentation too! However, this is currently are area
-we're severly lacking enough manpower to make great. If writing technical documentation is your area, we'd love your help!
+for lots of good documentation! However, we need a lot more help to write the
+awesome docs the project needs. If writing technical documentation is your area,
+we’d absolutely love your help!
+
+The best place to get started is by looking through the Github Issues at:
+https://github.com/ipfs/docs
 
-Best place to get started is by looking through the Github Issues over at: https://github.com/ipfs/docs
 
 ### Support
 
-IPFS already have lots of users and other curious people experimenting and using
-IPFS for their applications. These users sometimes get stuck or have questions
-that needs answering. If you've contributed something with code or documentation,
-chances are that you probably can help a lot of people with their questions.
+IPFS already has lots of users and curious people experimenting and using
+IPFS in their applications. These users sometimes get stuck or have questions
+that need answering. If you’ve contributed something with code or documentation,
+chances are that you can probably help a lot of people with their questions.
+
+We do most support via the forum we have at: https://discuss.ipfs.io/
 
-We mostly do support via the forum we have here: https://discuss.ipfs.io/
 
 ### Testing
 
-As we're continously improving IPFS everyday, mistakes can happen and we release
-something that could work much better or simply doesn't work. If you like to
-dig into edge-cases or writing testing scenarios, getting dirty with all our
-testing infrastructure would be a wet dream for you.
+We’re continously improving IPFS every day, but mistakes can happen and we
+could release something that doesn’t work as well as it should — or simply doesn't
+work at all! If you like to dig into edge-cases or write testing scenarios,
+wrangling our testing infrastructure could be the job for you.
 
-Mainly, CI work is happening over at https://github.com/ipfs/jenkins and larger
-scale tests are planned to be happening over at https://github.com/ipfs/kubernetes-ipfs
+We work on <abbr title="Continuous Integration">CI</abbr> tools at https://github.com/ipfs/jenkins
+and plan larger scale tests at https://github.com/ipfs/kubernetes-ipfs
 
 
 ### Design
 
-Currently, there is no fulltime designers working with IPFS and it's sister-projects.
-Although it would be nice to have, we still have many smaller pieces that can be
-contributed by you as a outsider!
+There are no full-time designers working on IPFS and its sister-projects, but
```

Comment:
I don't think this is true anymore. Maybe we can say "There is few full-time designers" instead.